robottelo/cli/imprt.py

```diff
@@ -0,0 +1,341 @@
+# -*- encoding: utf-8 -*-
+"""
+Usage::
+
+    hammer import [OPTIONS] SUBCOMMAND [ARG] ...
+
+Parameters::
+
+    SUBCOMMAND                    subcommand
+    [ARG] ...                     subcommand arguments
+
+Subcommands::
+
+    activation-key                Import Activation Keys
+                                  (from spacewalk-report activation-keys).
+    all                           Load ALL data from a specified directory
+                                  that is in spacewalk-export format.
+    config-file                   Create puppet-modules from Configuration
+                                  Channel content (from spacewalk-report
+                                  config-files-latest).
+    content-host                  Import Content Hosts
+                                  (from spacewalk-report system-profiles).
+    content-view                  Create Content Views based on local/cloned
+                                  Channels (from spacewalk-export-channels).
+    host-collection               Import Host Collections
+                                  (from spacewalk-report system-groups).
+    organization                  Import Organizations
+                                  (from spacewalk-report users).
+    repository                    Import repositories
+                                  (from spacewalk-report repositories).
+    repository-enable             Enable any Red Hat repositories
+                                  accessible to any Organization
+                                  (from spacewalk-report channels).
+    template-snippet              Import template snippets
+                                  (from spacewalk-report kickstart-scripts).
+    user                          Import Users (from spacewalk-report users).
+"""
+
+# import os.path
+from robottelo.cli.base import Base
+from robottelo.common import conf, ssh
+
+
+class Import(Base):
+    """Imports configurations from another Satellite instances"""
+
+    command_base = 'import'
+
+    @staticmethod
+    def prepare_import_data(tar_path=None):
+        """
+        Fetch and uncompress the CSV files from the source
+        """
+        # if path to tar file not specified, download the default one
+        if tar_path is None:
+            tar_remote_path = conf.properties['transitions.export_tar.url']
+            cmd = 'wget {0} -O - | tar -xzC /tmp'.format(tar_remote_path)
+        else:
+            cmd = 'tar -xzf {0} -C /tmp'.format(tar_path)
+
+        ssh.command(
+            cmd.encode('utf-8'), output_format='', timeout=30
```

Comment:
`s/cmd.encode('utf-8')/cmd` and add `u` before the cmd string.

Also the ssh.command call be be simplified to `ssh.command(cmd)`. This timeout is for the TCP connection and not for the duration of the command.